proposals/0004_changelog-strategy.md

```diff
@@ -87,16 +91,17 @@ Additional labels for pre-release and build metadata are available as extensions
 
 ### Rationale
 
-Unless [BEP-0002](https://github.com/brightway-lca/enhancement-proposals/blob/ca80e219917ed612f74936fd7f7e60cb3ee1b2eb/proposals/0002_merge-repositories.md) is accepted, the various Brightway packages will be maintained as separate repositories. To ensure the Sphinx documentation is able to pull changelog information from these repositories, they must follow a common naming and formatting convention.
+Unless BEP-0002 is accepted, the various Brightway packages will be maintained as separate repositories. To ensure the Sphinx documentation is able to pull changelog information from these repositories, they must follow a common naming and formatting convention.
 
 ### Pros and Cons
 
-Pros: Uniform formatting and structure across the changelog files of all Brightway packages. Improved readability, improved accountability of developers. Improved accessibility by end-users. \
-Cons: None
+Pros: Uniform formatting and structure across the changelog files of all Brightway packages. Improved readability, improved accountability of developers. Improved accessibility by end-users.
+
+Cons: None (!)
 
 ### Alternatives
 
-Maintain status quo, do not include changelog information in the documentation website.
+Maintain status quo, inconsistently include changelogs in the documentation but without a common naming and formatting convention.
 
```

Comment:
non-blocking: we could use [gnu style changelogs](https://www.gnu.org/prep/standards/html_node/Style-of-Change-Logs.html)